content/docs/user-guide/assets/asset-processor/configuration.md

```diff
@@ -5,21 +5,77 @@ description: Configure Asset Processor for Open 3D Engine (O3DE) with AssetProce
 weight: 300
 toc: true
 ---
+
+### Settings Registry
 
-You can modify the engine settings registry file for **Asset Processor** in `Registry/AssetProcessorPlatformConfig.setreg` to perform the following tasks:
+Many of the content processing rules that **Asset Processor** uses for your project are controlled by settings registry values. You can view the default values in [`Registry/AssetProcessorPlatformConfig.setreg`](https://github.com/o3de/o3de/blob/development/Registry/AssetProcessorPlatformConfig.setreg). This file is well commented, and full of examples you can use for specific configuration on your project's asset pipeline.
 
-* Add new source asset types for Asset Processor.
-* Update existing file type rules.
-* Update the ignore list.
-* Add or remove target platforms.
+{{< todo issue="https://github.com/o3de/o3de.org/issues/706" >}}
+To add project-specific or user-specific overrides, you can use the same pattern you would follow with other settings registry overrides. The linked ticket covers documenting the setttings registry override functionality.
+{{</todo>}}
 
-    {{< note >}}
-Asset Processor automatically builds assets for the host platform. For example, if Asset Processor is running on Windows, Asset Processor builds Windows assets even if pc is not enabled in the `.setreg` file.
-    {{< /note >}}
+### Processing Assets for Multiple Platforms
+
+To process assets for a non-host platform, you can enable additional platforms under the settings registry path `Amazon/AssetProcessor/Settings/Platforms`. Note that this is only one step in enabling a platform for your project. See the [platforms](/docs/user-guide/platforms/) section of the documentation for how to set up and deploy to additional platforms.

```

Comment:
Not quite too sure what you are trying to say. Are you saying there is more work to process assets for a non-host platform than just flipping the registry or are you saying "These settings only change the outputs of the Asset processor, to setup, build or deploy for additional platforms see..."?

Reply:
I'm trying to get at the idea that there are several steps involved in building and deploying to Android/iOS/other platforms. Enabling them for asset processing is one of those steps, but not the entire process.